### DISCUSSION TITLE
📚 [ Discussion ] Share Your Learning

### DISCUSSION THREAD

**Discussion by Kajol-Kumari** [HITK-TECH-Community/Community-Website#481] [Show and tell]
### P.S - Don't be SHY!! :eyes:

**Comment by vedikaag99** [comment]
I was a newbie to open source when I was introduced to this repository and trust me when I say this that I have learned a great deal since then. I, now, have a firm grip over git commands and also React JS apparently. It is not just writing codes that helped me but also learning new concepts all throughout and tackling different issues that pushed me to where I am right now. Thankful to the mentors for the consistent guidance. 😁❤️

**Comment by LeeRenJie** [comment]
DWOC was the first open-source program that I have taken part in! I have made 10 PRs in this particular repo and currently working on one more! From designing the logo of the website to making bug fixes, design improvement on the frontend of the website, and having my fixes merged one by one were very rewarding in the form of encouragement. This kept me to keep contributing. @Kajol-Kumari was very encouraging and patient to help me when I was facing issues while making beginner mistakes😅. I definitely improved my skills for React.js, open-source contribution git workflow,  CSS, SASS, and now I'm trying to figure out the JWT authentication for the website! Thanks for everything and the awesome community 😄

**Comment by jackfrost13** [comment]
> DWOC was the first open-source program that I have taken part in! I have made 10 PRs in this particular repo and currently working on one more! From designing the logo of the website to making bug fixes, design improvement on the frontend of the website, and having my fixes merged one by one were very rewarding in the form of encouragement. This kept me to keep contributing. @Kajol-Kumari was very encouraging and patient to help me when I was facing issues while making beginner mistakes😅. I definitely improved my skills for React.js, open-source contribution git workflow,  CSS, SASS, and now I'm trying to figure out the JWT authentication for the website! Thanks for everything and the awesome community 😄


We are glad to hear about your wonderful experience during the sprint and appreciate your efforts and sticking throughout the sprint. Feel free to leverage slack community channels if you have any doubts or want to share anything with others ❤️

**Comment by amanprateek123** [comment]
It was very interesting to work within this organisation. Learns a lot about the git tools and how the project workflow should be. Special thanks to @Kajol-Kumari  for consistently guiding us all.

**Comment by ashuvssut** [comment]
I would like to work on this issue 


😝

**Comment by ashuvssut** [comment]
I had an amazing time here. Thankyou @mentors for being so kind. This project is gonna help me become a good maintainer also as I am aiming to participate in an opensource event as a maintainer of my own React project. I am definitely gonna apply all good practices that I learnt by contributing this project

**Comment by sakshikhachane** [comment]
Starting from resolving branch conflicts to merging nearly 10 PRs, the journey was awesome. Initially, I was just aware of basic git commands for making PR but contributing to this project helped me gain knowledge about many git commands. Moreover, I am a newbie to React JS but this project gave me exposure to many concepts which helped me learn it quickly and in an easier way. Thanks to all the mentors who guided me throughout the program😄. Special thanks to @Kajol-Kumari for guiding me, helping me solve various basic to hard issues, and most important for the quick replies😅❤️.

**Comment by ashuvssut** [comment]
is this project taking part in an opensource event in this month?

**Reply by Kajol-Kumari** [reply]
Hey @ashuvssut we are taking part in GSSoC 2021

**Reply by ashuvssut** [reply]
I am working on my assigned issues that were assigned to me in SWOC n DWOC. please consider it for GSSOC

**Reply by jackfrost13** [reply]
Are you a GSSoC participant?

**Comment by ashuvssut** [comment]
Yes I am a GSSoC participant

**Comment by IryDev** [comment]
I learn to do responsive website using Bootstrap
